README.rst

```diff
@@ -0,0 +1,72 @@
+.. BADGES
+   ======
+.. image:: https://travis-ci.org/tgsmith61591/skutil.svg?branch=master
+   :target: https://travis-ci.org/tgsmith61591/skutil
+.. image:: https://coveralls.io/repos/github/tgsmith61591/skutil/badge.svg?branch=master
+   :target: https://coveralls.io/github/tgsmith61591/skutil?branch=master
+.. image:: https://img.shields.io/badge/python-2.7-blue.svg
+   :target: https://img.shields.io/badge/python-2.7-blue.svg
+.. image:: https://img.shields.io/github/release/tgsmith61591/skutil.svg
+   :target: https://img.shields.io/github/release/tgsmith61591/skutil
+.. image:: https://img.shields.io/github/issues-raw/tgsmith61591/skutil.svg
```

Comment:
I don't think I'm a fan of this. It's easily accessible and since we're using issues as a project mgmt tool, the inflated number just reflects a number of TODOs, not necessarily something bad. This kind of makes things looks worse than they are.